README.md

```diff
@@ -0,0 +1,78 @@
+# Moltimate Frontend
+
+Moltimate is an open source tool built to aid in protein function analysis. The project start with a group of student developers in the Software Engineering Department at Rochester Institute of Technology. Main dependencies for this project are  [React](https://reactjs.org/docs/getting-started.html) , [Material UI](https://material-ui.com/), [NGL View](https://github.com/arose/nglview).
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Must have Node version greater than 10 
+```
+node -v
+```
+If not installed then use the below command
+```
+brew install node
+```
+Checkout [n](https://github.com/tj/n) for Node version management
+
+If you have not installed Yarn before:
+```
+brew install yarn
+```
+
+### Installing
+
+To run locally you must download dependencies if it is your first time running the project, or anytime there is an addition to the packages.
+
+```
+yarn install
+```
+
+After the install
+
+```
+yarn start
+```
+
+If there are unit tests available to run:
+```
+yarn run test
+```
+
+Custom yarn scripts can be made such as `yarn build` which can be found in the `package.json`
+
+### Unit Tests
+
+Currently this project is not set up with any unit tests. This was done intentionally. Selenium driven integration tests unnecessary  increase developer time and do not provide useful checks for this software. This is built to simply display a protein visualization. Possible unit testable code is the advanced hook logic found in the `util` directory.
+
+## Project Structure
+
+Two core sections is the search form flow and the results page. Code is organized through actions, reducers, sagas, and components. Components are divided based on views. Containers are linked to the store and components are mostly stateless. 
+
+## Coding Style
+### General
+Use function based components when you can as they are lighter weight and performant
+Use Hooks and Context API 
+Generalize components as much as possible
+Put strings in the `strings.json` file at the top level of the directory and use throughout the project
+Do NOT use deprecated React Lifecycle methods such as `didComponentMount`
+If you are new to React take a comprehensive read of the below: 
+    Follow the [React Guidelines](https://reactjs.org/docs/hello-world.html) 
+    Follow [Hooks Guidelines](https://reactjs.org/docs/hooks-rules.html)
+
+### Style
+Following suggested linting from Airbnb
```

Comment:
Do we have linting enabled on the project with the Airbnb rules? Can this be included as a comment such as 
```suggestion
Follow suggested linting from Airbnb (Project using ESLint & rules: _link to AirBnb rules or place in repo where rules were copied to_)
```